src/Modules/SetName/Models/SetNameEntryModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel.DataAnnotations;
+using EvoSC.Manialinks.Attributes;
+
+namespace EvoSC.Modules.Official.SetName.Models;
+
+[EntryModel]
+public class SetNameEntryModel
+{
+    /// <summary>
+    /// The new desired nickname of the player.
+    /// </summary>
+    [MinLength(1)]
+    public string Nickname { get; set; }
```

Comment:
We might want to enforce stronger boundaries for names for example excluding newlines or exceptionally long names.

Reply:
Here are the relevant validation rules from current EvoSC (they're not great either)
https://github.com/EvoTM/EvoSC/blob/4d8faa96ef2ee0c98f17a2d736df36bfd4244f1e/core/Controllers/PlayerController.php#L103

Reply:
I made an issue for this as I think it goes beyond this pull request: https://github.com/EvoTM/EvoSC-sharp/issues/129